README.md

```diff
@@ -14,109 +14,98 @@ Heat is a distributed tensor framework for high performance data analytics.
-computational and memory needs of your laptop and desktop.
+# What is Heat for?
 
-# Features
+Heat builds on [PyTorch](https://pytorch.org/) and [mpi4py](https://mpi4py.readthedocs.io) to provide high-performance computing infrastructure for memory-intensive applications within the NumPy/SciPy ecosystem.
 
-* High-performance n-dimensional tensors
-* CPU, GPU, and distributed computation using MPI
-* Powerful data analytics and machine learning methods
-* Abstracted communication via split tensors
-* Python API
 
-# Support Channels
+With Heat you can:
+- port existing NumPy/SciPy code from single-CPU to multi-node clusters with minimal coding effort;
+- exploit the entire, cumulative RAM of your many nodes for memory-intensive operations and algorithms;
+- run your NumPy/SciPy code on GPUs (CUDA, ROCm, coming up: Apple MPS).
 
-We use [GitHub Discussions](https://github.com/helmholtz-analytics/heat/discussions) as a forum for questions about Heat.
-If you found a bug or miss a feature, then please file a new [issue](https://github.com/helmholtz-analytics/heat/issues/new/choose).
+Check out our [coverage tables](coverage_tables.md) to see which NumPy, SciPy, scikit-learn functions are already supported.
 
-# Requirements
+ If you need a functionality that is not yet supported:
+  - [search existing issues](https://github.com/helmholtz-analytics/heat/issues) and make sure to comment if someone else already requested it;
+  - [open a new issue](https://github.com/helmholtz-analytics/heat/issues/new/choose).
 
-Heat requires Python 3.7 or newer.
-Heat is based on [PyTorch](https://pytorch.org/). Specifically, we are exploiting
-PyTorch's support for GPUs *and* MPI parallelism. For MPI support we utilize
-[mpi4py](https://mpi4py.readthedocs.io). Both packages can be installed via pip
-or automatically using the setup.py.
 
-# Installation
+Check out our [features](#features) and the [Heat API Reference](https://heat.readthedocs.io/en/latest/autoapi/index.html) for a complete list of functionalities.
 
-Tagged releases are made available on the
-[Python Package Index (PyPI)](https://pypi.org/project/heat/). You can typically
-install the latest version with
+# Features
 
-```
-$ pip install heat[hdf5,netcdf]
-```
+* High-performance n-dimensional arrays
+* CPU, GPU, and distributed computation using MPI
+* Powerful data analytics and machine learning methods
+* Seamless integration with the NumPy/SciPy ecosystem
+* Python array API (work in progress)
 
-where the part in brackets is a list of optional dependencies. You can omit
-it, if you do not need HDF5 or NetCDF support.
+# Installation
 
-**It is recommended to use the most recent supported version of PyTorch!**
+## Requirements
 
-**It is also very important to ensure that the PyTorch version is compatible with the local CUDA installation.**
-More information can be found [here](https://pytorch.org/get-started/locally/).
+### Basics
+- python >= 3.8
+- MPI (OpenMPI, MPICH, Intel MPI, etc.)
```

Comment:
maybe sth that should be mentioned here but maybe this should also only bet mentioned in an "andmin section": 
* CUDA/ROCm installation compatible with your GPU, PyTorch-installation must match 
* for optimal performance on HPC-systems: MPI must be CUDA-aware (actually I dont know whether there is ROCm-aware MPI...)